### ISSUE TITLE
Issue installing 2.7.0

### ISSUE THREAD

**Issue by weotch** [born05/craft-twofactorauthentication#46]
When installing 2.7.0 for the first time I'm getting this error:

```
Exception: Class 'born05\twofactorauthentication\records\Session' not found (/Users/reinhard/Work/Clif Bar - Next/craft-cms/vendor/born05/craft-twofactorauthentication/src/migrations/Install.php:19)
#0 /Users/reinhard/Work/Clif Bar - Next/craft-cms/vendor/craftcms/cms/src/db/Migration.php(52): born05\twofactorauthentication\migrations\Install->safeUp()
#1 /Users/reinhard/Work/Clif Bar - Next/craft-cms/vendor/craftcms/cms/src/db/MigrationManager.php(232): craft\db\Migration->up(true)
#2 /Users/reinhard/Work/Clif Bar - Next/craft-cms/vendor/craftcms/cms/src/base/Plugin.php(140): craft\db\MigrationManager->migrateUp(Object(born05\twofactorauthentication\migrations\Install))
...
```

**Comment by roelvanhintum** [comment]
Sorry about that. I've fixed it in 2.7.2

**Comment by weotch** [comment]
Cool, that fixed it for me!
